Sources/PackageType/Package.swift

```diff
@@ -14,14 +14,16 @@ public class Package {
     public let url: String
     public let path: String
     public let name: String
+    public let isSourcePackage: Bool
     public var dependencies: [Package] = []
     public let manifest: Manifest
 
-    public init(manifest: Manifest, url: String) {
+    public init(manifest: Manifest, url: String, isSourcePackage: Bool = false) {
```

Comment:
I think we're standardizing on the name “root package”.

Aside: I think there is a better name that can be found.

But we can correct this later.